src/components/Clock.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component } from 'react';
+
+interface Props {
+  name: string;
+}
+
+interface State {
+  time: Date;
+}
+
+export class Clock extends Component<Props, State> {
+  state = {
+    time: new Date(),
+  };
+
+  intervalId = 0;
+
+  componentDidMount() {
+    this.intervalId = window.setInterval(() => {
+      // eslint-disable-next-line no-console
+      console.info(new Date().toUTCString().slice(-12, -4));
```

Comment:
create function for this `.toUTCString().slice(-12, -4)`. You will be able to reuse it